### PR TITLE
chore: add kokoro 3.9 configs

### PR DESCRIPTION
**Pull request by leahecole**
This will be blocked until 

- [x] [this synthtool PR](https://github.com/googleapis/synthtool/pull/909/files) is merged 
and 
- [ ] [we run synth here](https://github.com/googleapis/python-aiplatform/pull/6) (I rechecked the box to regen the PR on Friday 1/15 at 5:30pm pacific) 

Internal CL352076910 adds 3.6, 3.8, and 3.9 internal configs (3.7 is already set up)

### CODE REVIEW

**Comment by parthea** [comment]
Hey @leahecole,  Could we add the configurations to the synthtool templates [here](https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/python_library/.kokoro/samples)?

**Comment by parthea** [comment]
See synthtool feature request [here](https://github.com/googleapis/synthtool/issues/907).

**Comment by leahecole** [comment]
On it - thanks for the reference @parthea

**Comment by leahecole** [comment]
This will probably need a rework to instead specify the build specific project in a noxfile_config

**Comment by leahecole** [comment]
@parthea I was trying to follow through on my earlier comment and of course forgot what I meant. I added a noxfile_config to the model builder samples directory that calls the BUILD_SPECIFIC_GCLOUD_PROJECT - we define that value with the ucaip project in the kokoro configs. Given that kokoro configs are being sync'd with Owlbot, and we normally do a build specific project as one from our project pool, should I actually be doing this in the noxfile config?

What I think we need

```
'gcloud_project_env': 'string-of-project-name'
```

What we currently have (which references a project string defined in common.cfg

```
'gcloud_project_env': 'BUILD_SPECIFIC_GCLOUD_PROJECT'
```

**Comment by parthea** [comment]
Hey @leahecole ,

I'm not sure how to solve it with `noxfile_config`. In case it helps, this synthtool replacement in `owlbot.py` worked for me locally.

```
s.replace(".kokoro/samples/python3.*/common.cfg",
    """env_vars: \{
    key: "BUILD_SPECIFIC_GCLOUD_PROJECT"
    value: "python-docs-samples-tests-py3.*?"
\}""",
    """env_vars: {
    key: "BUILD_SPECIFIC_GCLOUD_PROJECT"
    value: "ucaip-sample-tests"
}""",
    flags=re.MULTILINE | re.DOTALL,
)
```

**Comment by parthea** [comment]
Hey @leahecole, 
Please let me know if I can go ahead with adding the replacement in owlbot.py from https://github.com/googleapis/python-aiplatform/pull/175#issuecomment-881470103

**Comment by leahecole** [comment]
That will work and I'm happy to hand it off to you!

**Comment by parthea** [comment]
@googlebot I consent.

**Comment by parthea** [comment]
@leahecole Done,please can you take a look?

File: .kokoro/samples/python3.6/periodic-head.cfg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+env_vars: {
+    key: "INSTALL_LIBRARY_FROM_SOURCE"
+    value: "True"
+}
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/python-pubsub/.kokoro/test-samples-against-head.sh"
```
Comment (leahecole) [inline .kokoro/samples/python3.6/periodic-head.cfg:10]:
```suggestion
    value: "github/python-aiplatform/.kokoro/test-samples-against-head.sh"
```

File: .kokoro/samples/python3.7/periodic-head.cfg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+env_vars: {
+    key: "INSTALL_LIBRARY_FROM_SOURCE"
+    value: "True"
+}
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/python-pubsub/.kokoro/test-samples-against-head.sh"
```
Comment (leahecole) [inline .kokoro/samples/python3.7/periodic-head.cfg:10]:
```suggestion
    value: "github/python-aiplatform/.kokoro/test-samples-against-head.sh"
```

File: .kokoro/samples/python3.8/periodic-head.cfg [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+env_vars: {
+    key: "INSTALL_LIBRARY_FROM_SOURCE"
+    value: "True"
+}
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/python-pubsub/.kokoro/test-samples-against-head.sh"
```
Comment (leahecole) [inline .kokoro/samples/python3.8/periodic-head.cfg:10]:
```suggestion
    value: "github/python-aiplatform/.kokoro/test-samples-against-head.sh"
```

File: .kokoro/samples/python3.9/periodic-head.cfg [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+env_vars: {
+    key: "INSTALL_LIBRARY_FROM_SOURCE"
+    value: "True"
+}
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/python-pubsub/.kokoro/test-samples-against-head.sh"
```
Comment (leahecole) [inline .kokoro/samples/python3.9/periodic-head.cfg:10]:
```suggestion
    value: "github/python-aiplatform/.kokoro/test-samples-against-head.sh"
```

**Comment by leahecole** [comment]
@parthea other than these little fixes (which very well could have been a copy paste error from me) it looks good!

File: .kokoro/samples/python3.6/periodic-head.cfg (hunk 1 above)
Comment (parthea) [reply .kokoro/samples/python3.6/periodic-head.cfg:10]: Great catch! It's strange to see pubsub here. I was working with pubsub yesterday but I'm not sure how the changes landed here. [This is the commit](https://github.com/googleapis/python-aiplatform/commit/a0cbbe09ba036591ff4c24c2c623b05928667b84) where the changes were added. I'm looking into how it happened.

Comment (parthea) [reply .kokoro/samples/python3.6/periodic-head.cfg:10]: It looks like python-pubsub is hardcoded in synthtool templates here: https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/.kokoro/samples/python3.6/periodic-head.cfg#L10

Comment (parthea) [reply .kokoro/samples/python3.6/periodic-head.cfg:10]: Will be fixed in https://github.com/googleapis/synthtool/pull/1167

File: .kokoro/samples/python3.7/periodic-head.cfg (hunk 2 above)
Comment (parthea) [reply .kokoro/samples/python3.7/periodic-head.cfg]: Fixed in https://github.com/googleapis/python-aiplatform/pull/175/commits/437b5a138ef135353b08d12ae1a217924d4cacfe

File: .kokoro/samples/python3.8/periodic-head.cfg (hunk 3 above)
Comment (parthea) [reply .kokoro/samples/python3.8/periodic-head.cfg]: Fixed in https://github.com/googleapis/python-aiplatform/pull/175/commits/437b5a138ef135353b08d12ae1a217924d4cacfe

File: .kokoro/samples/python3.9/periodic-head.cfg (hunk 4 above)
Comment (parthea) [reply .kokoro/samples/python3.9/periodic-head.cfg]: Fixed in https://github.com/googleapis/python-aiplatform/pull/175/commits/437b5a138ef135353b08d12ae1a217924d4cacfe

File: .kokoro/samples/python3.6/periodic-head.cfg (hunk 1 above)
Comment (parthea) [reply .kokoro/samples/python3.6/periodic-head.cfg]: Fixed in https://github.com/googleapis/python-aiplatform/pull/175/commits/437b5a138ef135353b08d12ae1a217924d4cacfe

**Review by leahecole — COMMENTED** [review]
LGTM!

**Comment by leahecole** [comment]
Because I'm the original author I can't approve so perhaps another @googleapis/python-samples-owners can approve :)

**Review by busunkim96 — APPROVED** [review]

**Comment by leahecole** [comment]
Thanks @busunkim96 :)

**Review by parthea — APPROVED** [review]

**Pull request merged**